robot/robot.py

```diff
@@ -117,7 +119,7 @@ def teleopPeriodic(self):
         if self.right_joystick.getTrigger() or self.fire_toggled:
             self.shooter_control.fire()
         elif self.right_joystick.getRawButton(2) or self.ds_ball_in:
```

Comment:
autolenny should be enabled/disabled by an NT variable only. The joystick should still call ball_in. 

Consider enabling autolenny by default -- there are arguments for/against this.